bff/pkg/server/server.go

```diff
@@ -358,6 +358,11 @@ func activity_handler(metrics_service metrics.Service) http.HandlerFunc {
 			"dim":   dim,
 		}
 
+		if limitLegend != "" {
```

Comment:
question: for the API user, are they passing "limitlegend" or "limit_legend"?

Reply:
limitlegend. I chose that because thats the format other params are in. If we want another format let me know